amazon-cloudwatch-container-insights/k8s-deployment-manifest-templates/deployment-mode/service/cwagent-prometheus/prometheus-eks.yaml

```diff
@@ -47,8 +50,8 @@ data:
                   ]
                 },
                 {
-                  "source_labels": ["Service", "backend"],
-                  "label_matcher": ".*haproxy-ingress-.*metrics;(httpback-shared-backend|httpback-default-backend|httpsback-shared-backend|_default_backend|.*backend_http)",
+                  "source_labels": ["Service"],
```

Comment:
Would this be updated in [cloudwatch container insights repository](https://github.com/aws/amazon-cloudwatch-agent/tree/master/amazon-cloudwatch-container-insights) and also in public documents?

Reply:
https://github.com/aws-samples/amazon-cloudwatch-container-insights/pull/97/files it's going to be there, need someone to approve it.